modules/oe_content_event/config/install/field.storage.node.field_featured_media_legend.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_featured_media_legend
+field_name: field_featured_media_legend
+entity_type: node
+type: string
+settings:
+  max_length: 255
```

Comment:
should be 150

Reply:
Done through max length module inside entity form display config